### PR TITLE
Added breakpoints for user input for the player

### LINKED ISSUE #520: (re)design the current played song UI
I don't know if this is possible for you, but it would be great if we could get something like the big circle shown on small portrait screens ?
I often listen to music from a Rpi with the official 7" touchscreen display, I access the webUI and everything is ok.
But I would love to have bigger multimedia controls (play/pause, ffw/fbw, rating,...) and a view of the album art corresponding.

### PR DESCRIPTION
**Pull request by the-pro**
Resolves #520 

Added the input for the user to give the appropriate breakpoints for the player and the player now works on those inputs

### CODE REVIEW

**Comment by jvoisin** [comment]
Can you please add screenshots, so everyone can see what your changes are doing?

**Comment by the-pro** [comment]
Yeah sure

I added a select list of breakpoints for users to select according to their screen size and choice :
Screenshot from 2021-03-16 17-24-49

If the break point is set >= their screen size then it will change the player :

Screenshot from 2021-03-16 17-25-37

**Review by deluan — CHANGES_REQUESTED** [review]
Thanks for this, please take a look at the comments below. Also, I found two visual issues:
1. When first loading the UI with your changes, there's no default value, and the behaviour is unexpected:
Screen Shot 2021-03-16 at 11 52 03 PM

and the style of the input is wrong (does not match the other inputs:
Screen Shot 2021-03-16 at 11 50 44 PM

File: ui/src/personal/Personal.js [hunk 1]
```diff
@@ -103,6 +108,29 @@ const SelectTheme = (props) => {
...
+  const currentResolution = useSelector((state) => state.settings.resolution)
+  const dispatch = useDispatch()
+  const resChoices = Object.keys(resolution).map((key) => {
+    return { id: key, name: resolution[key].value }
+  })
+
+  return (
+    <SelectInput
+      {...props}
+      label={'select mobile player resolution'}
```
[the remark this answers is not in the document]
Comment (the-pro) [reply ui/src/personal/Personal.js]: I have added the translate function and configured the language file accordingly but it does not work on any other translation than english

[the remark this answers is not in the document]
Comment (deluan) [reply ui/src/personal/Personal.js]: That's fine. This function does not translate automatically. When/If this PR is merged, we can receive contributions to the [translation files](https://github.com/navidrome/navidrome/tree/fbe6ecea95c153cc520386291e95df3f0b11b170/resources/i18n)

**Comment by deluan** [comment]
I think the UX is still not great.... I'll have to spend some time trying to sort it out, and then will come back to this with some feedback. Thanks for your work on this so far.

PS: There may be some wrong usage of Redux somewhere in your code. I see this message in the browser's console if I start an incognito session:
Screen Shot 2021-03-21 at 1 51 21 PM

**Comment by the-pro** [comment]
I am thinking of storing it right in the local storage directly. Would that be alright

**Comment by the-pro** [comment]
sir @deluan could you have a look at this commit. I have changed the use of redux store to localStorage.

**Comment by deluan** [comment]
Sorry for the late reply. We have to avoid using localStorage as much as we can. Specially because in the future Preferences will go to the server and if it is all in Redux, we can just implement something like a Redux Saga to send the data to the server.

I know that handling Redux can be overwhelming at first, but I think this is a good opportunity for you to learn more about it, as it is one of the most used frameworks to handle state in React.

**Comment by deluan** [comment]
Hey, I'm closing this as I'll need to rewrite the UI with the new React-Admin version (v4) and will try to implement this in a different way. Thanks anyway for the contribution!

**Pull request closed without merging**